source/SpinalHDL/Developers area/mill support.rst

```diff
@@ -0,0 +1,47 @@
+Build through Mill
+==================
+
+SpinalHDL itself can be built by Mill build tools, it could compile/test/publishLocal the existing modules.
+Build through mill can be much faster than sbt, which is useful while debugging.
+
+Complie the library
+-------------------
+
+.. code-block:: sh
+
+   mill __.compile
+   sbt compile # equivalent alternatives
```

Comment:
So it doesn't compile tests?

Reply:
no, that's why its name is "compile the library"